Scripts/git_hooks/pre-commit

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
```

Comment:
This is just a straight copy of my existing script, which I'm pretty sure is untouched from what you provided @charlesmchen.

Reply:
Ah! Do you mind if I move it over top of this one? 

For dev setup, it would be nice if these were in one place.

Reply:
>  Can we de-dupe somehow?

Done! PTAL @charlesmchen